iina/MainWindowController.swift

```diff
@@ -550,6 +550,19 @@ class MainWindowController: PlayerWindowController {
     // gesture recognizer
     cv.addGestureRecognizer(magnificationGestureRecognizer)
 
+    // Workaround a bug in macOS Ventura where HDR content becomes dimmed when playing in full
+    // screen mode once overlaying views are fully hidden (issue #3844). After applying this
+    // workaround another bug in Ventura where an external monitor goes black could not be
+    // reproduced (issue #4015). The workaround adds a tiny subview with such a low alpha level it
+    // is invisible to the human eye.
```

Comment:
I'd like a short comment mentioning that this workaround isn't perfect

Reply:
Added note that it might not be effective in all cases.